src/dataframe/dataframe.jl

```diff
@@ -278,6 +278,26 @@ _columns(df::DataFrame) = getfield(df, :columns)
 nrow(df::DataFrame) = ncol(df) > 0 ? length(_columns(df)[1])::Int : 0
 ncol(df::DataFrame) = length(index(df))
 
+##############################################################################
+##
+## DataFrame consistency check
+##
+##############################################################################
+
+function _check_consistency(df::DataFrame)
```

Comment:
I don't think we generally use underscores for private functions, do we?

Reply:
Not always, but I think we should. This is a more clear signal between a function that is not exported and really is internal (like this one) and a function that could be potentially exported or used by the end user (like `hcat!`).